gubernator/filters.py

```diff
@@ -62,6 +63,21 @@ def rep(m):
                                 flags=re.MULTILINE))
 
 
+def do_testcmd(name):
+    if name.startswith('k8s.io/'):
+        if name.count(' ') != 1:
```

Comment:
It's how they're labeled in the junit.xml files -- this is mostly a guard to keep an error on the next line from crashing the whole page.